oidc/client.go

```diff
@@ -66,11 +66,16 @@ type ClientConfig struct {
 }
 
 func NewClient(cfg ClientConfig) (*Client, error) {
+	ru, err := phttp.ParseNonEmptyURL(cfg.RedirectURL)
```

Comment:
Clients need a callback URL in order to get tokens from the OIDC server. That's the only way a token can be obtained. Why would a client ever need to validate a token without allowing it to be generated?

Are there any actual use-cases requiring verification without generation? Even if a user is pasting in a token + refresh token, I think the client still needs to support logging in via the browser and the 3-legged OAuth2 flow if available.